### PR TITLE
update genometools to 1.5.10

### PR DESCRIPTION
**Pull request by notestaff**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/genometools-genometools/meta.yaml [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
+  number: 0
 
 requirements:
   build:
-    - gcc
+    - gcc  # [not osx]
+    - llvm  # [osx]
     - python
-    - pango
+    - pango  # [not osx]
```
Comment (bgruening) [inline recipes/genometools-genometools/meta.yaml]: out of interest why does cairo not work for OSX?

Comment (notestaff) [reply recipes/genometools-genometools/meta.yaml]: Cairo works, but we'd also need xquartz, which isn't currently in conda-forge:
https://github.com/genometools/genometools/blob/master/INSTALL#L51

In general, is this the right way to provide build variants (here one not requiring cairo/pango/X) of recipes, as subdirs of the main recipe, with the same version but different build string?  Conda also has 'features' but hardly any recipe seems to use that.

Comment (bgruening) [reply recipes/genometools-genometools/meta.yaml]: features are really broken and should not be used. I'm fine with merging it, was just curious.

**Comment by standage** [comment]
It would be awesome if AnnotationSketch were available on the OSX version of this package. It looks like @epruesse has a list [here](https://github.com/conda-forge/staged-recipes/issues/2587#issuecomment-293111507) of remaining obstacles to integrating XQuartz into conda forge. Before I start getting my hands dirty with this, do you know if any progress has been made on that list since then, @epruesse?

**Pull request merged**